SQLTools.py

```diff
@@ -132,7 +133,10 @@ def getSelection():
     if View().sel():
         for region in View().sel():
             if region.empty():
-                text.append(View().substr(View().line(region)))
+                if not settings.get('expand_to_paragraph', False):
```

Comment:
it's added in the next commit, https://github.com/optio/SQLTools/commit/146141e827e232f39bc42928e781ca831bf25f05 .